l10n_eu_product_adr_report/report/product_adr_report.xml

```diff
@@ -1,91 +1,90 @@
                 <t t-foreach="docs" t-as="o">
-                    <img t-if="o.company_id.logo" t-att-src="'data:image/png;base64,%s' % o.company_id.logo" style="max-height:80px;"/>
-                    <br/>
-                    <br/>
-                    <br/>
-                    <t t-foreach="o.move_lines" t-as="p">
-                        <table>
-                            <tr>
-                                <td><b>Shipper: </b></td>
-                                <td style="padding: 30px;"><span t-field="o.company_id" /><span t-field="o.company_id.street" /><br/>
-                                    <span t-field="o.company_id.zip" /><span t-field="o.company_id.city" /><br/>
-                                    tel:<span t-field="o.company_id.phone" /></td>
-                            </tr>
-                            <tr>
-                                <td><b>Picking number: </b></td>
-                                <td style="padding-left: 15px;"><span t-field="o.name" /></td>
-                            </tr>
-                            <tr>
-                                <td><b>Code: </b></td>
-                                <td style="padding-left: 15px;"><span t-field="p.product_id" /></td>
-                            </tr>
-                            <tr>
-                                <td><b>Quanty: </b></td>
-                                <td style="padding-left: 15px;"><span t-field="p.product_qty" /></td>
-                            </tr>
-                            <tr>
-                                <td><b>Class: </b></td>
-                                <td style="padding-left: 15px;"><span t-field="p.product_id.adr_class_id.name" /></td>
-                            </tr>
-                            <tr t-if="p.product_id.adr_class_id.picking_text">
-                                <td><b>Material: </b></td>
-                                <td style="padding-left: 15px;"><span t-field="p.product_id.adr_class_id.picking_text" /></td>
-                            </tr>
-                            <tr>
-                                <td><b>Date: </b></td>
-                                <td style="padding-left: 15px;"><span t-field="p.date" t-field-options='{"format": "dd/MM/yyyy"}'/></td>
-                            </tr>
-                            <tr>
-                                <td><b>Vehicle registration:</b></td>
-                                <td style="padding-left: 15px;"></td>
-                            </tr>
-                            <tr>
-                                <td><b>Departure time:</b></td>
-                                <td style="padding-left: 15px;"></td>
-                            </tr>
-                        </table>
+                    <t t-call="web.external_layout">
+                        <img t-if="o.company_id.logo" t-att-src="'data:image/png;base64,%s' % o.company_id.logo" style="max-height:80px;"/>
```

Comment:
Where is this logo appearing? As you are already using the external layout, maybe this is not longer needed.